app/views/configurations/index.html.erb

```diff
@@ -2,9 +2,14 @@
 
 <div class="row-fluid">
   <div class="span7 offset1">
-    <%= form_tag configurations_path do %>
-      <% @configs.each do |configuration| %>
-        <%= render partial: configuration[:type], object: configuration %>
+    <% if @configs.count == 0 %>
+      <h2>This page intentionally left blank</h2>
```

Comment:
@etdsoft Added more information and removed the "This page intentionally left blank" line. Lmk what you think.